packages/api-server/src/requestHandlers/awsLambdaFastify.ts

```diff
@@ -39,6 +48,57 @@ const lambdaEventForFastifyRequest = (
   } as APIGatewayProxyEvent
 }
 
+/**
+ * The header keys are case insensitive, but Fastify prefers these to be lowercase.
+ * Therefore, we want to ensure that the headers are always lowercase and unique
+ * for compliance with HTTP/2.
+ *
+ * @see: https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2
+ *
+ * Just as in HTTP/1.x, header field names are strings of ASCII
+ * characters that are compared in a case-insensitive fashion.  However,
+ * header field names MUST be converted to lowercase prior to their
+ * encoding in HTTP/2.  A request or response containing uppercase
+ * header field names MUST be treated as malformed (Section 8.1.2.6).
+ *
+ * @see: https://tools.ietf.org/html/rfc2616#section-4.2
+ */
+const sanitizeHeaders = (
```

Comment:
As explained in the fastify issue - they've implemented this as a performance optimisation to only support lower case headers. Why do we need to sanitise them here? Just adds extra loops right?

Reply:
DT will remove the PascalCase ContentType from our graphql function, and dbAuth.